src/AgentHeaderDescriptor.php

```diff
@@ -40,6 +40,7 @@
 class AgentHeaderDescriptor
 {
     const AGENT_HEADER_KEY = 'x-goog-api-client';
+    const GAX_VERSION = '0.5.0-dev';
```

Comment:
Google cloud PHP has a tool to semi automate the version bump. If you like I can help you with something similar for GAX.